### PR TITLE
H2planes fix

### PR DESCRIPTION
**Pull request by MariannaR**

### CODE REVIEW

File: inst/extdata/inconv_factor.csv [hunk 1]
```diff
@@ -382,6 +382,7 @@ SSP5,Mix4,targetValBEV,0
 SSP5,Mix4,startYeICE,2020
 SSP5,Mix4,targetYeICE,2027
 SSP5,Mix4,targetValICE,0.2
+SSP5,Mix4,ratioPHEV,0.5
```
Comment (Loisel) [inline inst/extdata/inconv_factor.csv:385]: This change seems unrelated.

**Review by Loisel — APPROVED** [review]
Looks good. And that does it?

File: inst/extdata/inconv_factor.csv (hunk 1 above)
Comment (MariannaR) [reply inst/extdata/inconv_factor.csv:385]: this is from the merging of `pik-piam/master`, you are right it is unrelated

**Comment by MariannaR** [comment]
prices and shares out of the logit calculations are included for H2 airplanes correctly. I did not look at the validation plots yet.

**Review by johannah-pik — APPROVED** [review]
